includes/class-micropub-render.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+// Post Content Filter
+add_filter( 'micropub_post_content', array( 'Micropub_Render', 'generate_post_content' ), 1, 2 );
+
+
+/**
+ * Micropub Render Class
+ */
+class Micropub_Render {
+	/**
+	 * Generates and returns a post_content string suitable for wp_insert_post()
+	 * and friends.
+	 */
+	public static function generate_post_content( $post_content, $input ) {
```

Comment:
this function and `generate_event` are still in `class-micropub-endpoint.php`. they shouldn't be, right? mind removing them from there?